### DISCUSSION TITLE
Are there any latency requirements for validators

### DISCUSSION THREAD

**Discussion by ok-ul-ch** [MystenLabs/sui#16665] [Q&A]
As mentioned [here](https://blog.sui.io/sui-performance-update/) Sui's TTF is ~480ms. 

Are there any implicit or explicit network latency requirements to validators so that TTF can be that low?
 
What if I will run a validator that has, let's say, 1s latency to other validators in the network? Is there a risk for me to get slashed due to high latency?

**Comment by mwtian** [comment]
Yes a validator with 1s latency to all other peers will have very poor consensus block proposal performance. Eventually it will get penalized by the network e.g. receiving no rewards. Also the Sui consensus tolerates up to 20% stake of poor performing or down validators without the latency getting affected.
